### PR TITLE
Fix wrong terminology used in place of Log Update

### LINKED ISSUE #7350: "Log Update"- consistency in terminology
**Is your feature request related to a problem? Please describe.**
Log update is mentioned as "Daily rounds" and "consultation updates" in various places. 

**Describe the solution you'd like**

- [ ] In patient dashboard (https://care.coronasafe.in/facility/0c95c7f0-e1d2-4aff-83fa-933cef60d3a8/patient/7e8fc20f-02da-47f7-8c33-f2d8c7e52dd8/consultation/cb44a173-6fe2-452f-b2d1-6c6e2695411e) 

Screenshot 2024-03-06 at 4 52 50 PM

- [ ]  In patient dashboard (https://care.coronasafe.in/facility/0c95c7f0-e1d2-4aff-83fa-933cef60d3a8/patient/7e8fc20f-02da-47f7-8c33-f2d8c7e52dd8/consultation/cb44a173-6fe2-452f-b2d1-6c6e2695411e) 

Screenshot 2024-03-06 at 4 54 05 PM

Screenshot 2024-03-06 at 4 55 33 PM

- [ ] viewing a saved log update (https://care.coronasafe.in/facility/0c95c7f0-e1d2-4aff-83fa-933cef60d3a8/patient/7e8fc20f-02da-47f7-8c33-f2d8c7e52dd8/consultation/cb44a173-6fe2-452f-b2d1-6c6e2695411e/daily_rounds/2da792c7-7ff8-4b8f-8e68-1397b7848597)
title shows "consultation update"

- [ ] https://care.coronasafe.in/facility/0c95c7f0-e1d2-4aff-83fa-933cef60d3a8/patient/7e8fc20f-02da-47f7-8c33-f2d8c7e52dd8/consultation/cb44a173-6fe2-452f-b2d1-6c6e2695411e/daily-rounds/6469a83e-470e-4fbb-a8ee-51770a42360e Viewing a saved log update, title shows "Consultation update" and bread crumb shows "Daily rounds"

Change all the above to "Log Update"

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by yash-learner**
## Proposed Changes

- Closes #7350 

> Things to check:

- Changes related to events and daily_rounds is left out as events will be deprecated
- Changing description requires backend change - Need to check whether a backend PR is required as changing the message would be simple but changing constants PATIENT_CONSULTATION_UPDATE_CREATED to PATIENT_LOG_UPDATE_CREATED requires a migration and also we may revamp this DailyRounds table to something else future based on FHIR 

This pull request includes significant updates to the application's routing and terminology, specifically changing references from "daily rounds" to "log updates". The changes span multiple files and components, ensuring consistency across the codebase.

Key changes include:

### Routing and Terminology Updates:
* [`src/Routers/routes/ConsultationRoutes.tsx`](diffhunk://#diff-7f7cd3879325257f8c0b086345142dc4e8e501dab940ac79820595ccb9a02816L84-R84): Updated various routes to replace "daily-rounds" with "log_updates" for patient consultations. [[1]](diffhunk://#diff-7f7cd3879325257f8c0b086345142dc4e8e501dab940ac79820595ccb9a02816L84-R84) [[2]](diffhunk://#diff-7f7cd3879325257f8c0b086345142dc4e8e501dab940ac79820595ccb9a02816L95-R95) [[3]](diffhunk://#diff-7f7cd3879325257f8c0b086345142dc4e8e501dab940ac79820595ccb9a02816L104-R104) [[4]](diffhunk://#diff-7f7cd3879325257f8c0b086345142dc4e8e501dab940ac79820595ccb9a02816L113-R113) [[5]](diffhunk://#diff-7f7cd3879325257f8c0b086345142dc4e8e501dab940ac79820595ccb9a02816L122-R122)
* [`src/common/constants.tsx`](diffhunk://#diff-0052448600cc7397e734b7c05fdf86b5b63c9035c531fc3b7c798d02d7c991feL656-R661): Modified notification event text to reflect the new terminology.
* [`src/components/Facility/ConsultationCard.tsx`](diffhunk://#diff-5a806f7f1e935e47668c590a3311adc31c8752505f16bb225c69bbe20fc9307dL203-R203): Updated navigation paths to use "log_updates" instead of "daily-rounds".
* [`src/components/Facility/ConsultationDetails/ConsultationUpdatesTab.tsx`](diffhunk://#diff-73c7edd39ba20073cb7581c8dadbdebdb1a0e332b2101dd2987112c2c5ba3b5fL691-R691): Changed tab text from "daily_rounds" to "log_updates".
* [`src/components/Facility/Consultations/DailyRounds/DefaultLogUpdateCard.tsx`](diffhunk://#diff-0125914f62dc3efdf0991dfadacf335d4979279e888bdcaa1253f4b9ac4b5a83L62-R63): Updated URLs in the `DefaultLogUpdateCard` component. [[1]](diffhunk://#diff-0125914f62dc3efdf0991dfadacf335d4979279e888bdcaa1253f4b9ac4b5a83L62-R63) [[2]](diffhunk://#diff-0125914f62dc3efdf0991dfadacf335d4979279e888bdcaa1253f4b9ac4b5a83L84-R85)

### Component Updates:
* [`src/components/LogUpdate/CriticalCareEditor.tsx`](diffhunk://#diff-1b3f7dbec2947f38d3fbde7ce367aba4b6a5d0d020cdc94b2f9c2abbf7e7fab8L98-R98): Adjusted links and state variables to use "log_updates". [[1]](diffhunk://#diff-1b3f7dbec2947f38d3fbde7ce367aba4b6a5d0d020cdc94b2f9c2abbf7e7fab8L98-R98) [[2]](diffhunk://#diff-1b3f7dbec2947f38d3fbde7ce367aba4b6a5d0d020cdc94b2f9c2abbf7e7fab8L181-R181)
* [`src/components/LogUpdate/CriticalCarePreview.tsx`](diffhunk://#diff-34be1b2cccb7da44958b357183af90823b705362062ffd39d9f9e33105c4767aL64-R64): Updated text to use the new terminology.
* [`src/components/Notifications/NotificationsList.tsx`](diffhunk://#diff-452b306140bd79c85c2fe84b34c51892918cf7e3956f39d6f956a72fb936366cL75-R77): Modified notification URLs.
* [`src/components/Notifications/ShowPushNotification.tsx`](diffhunk://#diff-6795d24944a45978b1d1cc6529714751fb9f0c048bb70ccb82aacf68991a3f43L27-R29): Updated paths for push notifications.
* [`src/components/Patient/DailyRoundListDetails.tsx`](diffhunk://#diff-1edfade0c155086d520f0487d31469a48a0eb401b2595383505e83a9e7c95d25L34-R35): Changed titles and back URLs to reflect "log_updates". [[1]](diffhunk://#diff-1edfade0c155086d520f0487d31469a48a0eb401b2595383505e83a9e7c95d25L34-R35) [[2]](diffhunk://#diff-1edfade0c155086d520f0487d31469a48a0eb401b2595383505e83a9e7c95d25L54-R54)
* [`src/components/Patient/DailyRounds.tsx`](diffhunk://#diff-2404028de0ea08f7854fcc9106d5e778e59fa44496771088f5dc0a1b67f6f51aL149-R149): Updated navigation paths and text. [[1]](diffhunk://#diff-2404028de0ea08f7854fcc9106d5e778e59fa44496771088f5dc0a1b67f6f51aL149-R149) [[2]](diffhunk://#diff-2404028de0ea08f7854fcc9106d5e778e59fa44496771088f5dc0a1b67f6f51aL416-R416) [[3]](diffhunk://#diff-2404028de0ea08f7854fcc9106d5e778e59fa44496771088f5dc0a1b67f6f51aL444-R444) [[4]](diffhunk://#diff-2404028de0ea08f7854fcc9106d5e778e59fa44496771088f5dc0a1b67f6f51aL558-R558)
* [`src/components/Patient/PatientInfoCard.tsx`](diffhunk://#diff-bb28030f49470d250f84f0896797579e57502fd3dea2a7a28ec261dbaa7180d9L585-R585): Adjusted links for patient consultations.

### Localization:
* [`public/locale/en.json`](diffhunk://#diff-d120f97dc8f2f2c516a2ea69a046e008947dbbc26e5a77467a102837c665f923R825-R826): Added new entries for "log_update" and "log_updates".
- Change 1
- Change 2
- More?

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

## Release Notes

- **New Features**
  - Enhanced localization support for logging functionalities, including new entries for log updates.
  - Updated navigation paths and button functionalities to reflect the new "log_updates" terminology.

- **Bug Fixes**
  - Improved error handling and notifications related to patient bed assignments.

- **Refactor**
  - Renamed components and updated import paths to align with the new logging structure.
  - Refactored test cases to ensure consistency with updated UI elements.

- **Documentation**
  - Updated component titles and button labels for better user experience and clarity.

### CODE REVIEW

**Comment by yash-learner** [comment]
> Things to check:

- [x] Changes related to events and daily_rounds is left out as events will be deprecated
- [x] Changing description of notification requires backend change - Need to check whether a backend PR is required as changing the message would be simple but changing constants `PATIENT_CONSULTATION_UPDATE_CREATED` to `PATIENT_LOG_UPDATE_CREATED` requires a migration and also we may revamp this DailyRounds table to something else future based on FHIR.

**Review by rithviknishad — DISMISSED** [review]

File: src/components/Scribe/formDetails.ts [hunk 1]
```diff
@@ -360,9 +360,9 @@ const DAILY_ROUND_FORM_SCRIBE_DATA: Field[] = [
 ];
 
 export const SCRIBE_FORMS: { [key: string]: ScribeForm } = {
-  daily_round: {
-    id: "daily_round",
-    name: "Daily Round",
+  log_update: {
+    id: "log_update",
+    name: "Log Update",
```
[the remark this answers is not in the document]
Comment (yash-learner) [reply src/components/Scribe/formDetails.ts]: @coderabbitai for now we are only changing the text in UI and routes and not the code in fe and backend as we will soon revamp it.

[the remark this answers is not in the document]
Comment (yash-learner) [reply src/components/Scribe/formDetails.ts]: @coderabbitai FYI, we recently moved the SCRIBE related code in DailyRounds component and formDetails as a plug. so previous change I made in formDetails are no longer valid after I merged latest develop branch

File: src/components/Notifications/ShowPushNotification.tsx [hunk 2]
```diff
@@ -24,9 +24,9 @@ export default function ShowPushNotification({ id }: { id: string }) {
       case "PATIENT_CONSULTATION_UPDATED":
         return `/facility/${caused_objects?.facility}/patient/${caused_objects?.patient}/consultation/${caused_objects?.consultation}`;
       case "PATIENT_CONSULTATION_UPDATE_CREATED":
-        return `/facility/${caused_objects?.facility}/patient/${caused_objects?.patient}/consultation/${caused_objects?.consultation}/daily-rounds/${caused_objects?.daily_round}`;
+        return `/facility/${caused_objects?.facility}/patient/${caused_objects?.patient}/consultation/${caused_objects?.consultation}/log_updates/${caused_objects?.daily_round}`;
```
[the remark this answers is not in the document]
Comment (yash-learner) [reply src/components/Notifications/ShowPushNotification.tsx:27]: We still call the backend model as DailyRound and not opening an PR as we will soon revamping it according FHIR

> src/Utils/request/api.tsx

Backend api routes 

> src/components/Notifications/ShowPushNotification.tsx

`/log_updates/${caused_objects?.daily_round` -> this as well coming from backend 

> src/components/Patient/ManagePatients.tsx

This as well from backend

> src/components/Patient/DailyRounds.tsx && src/components/Scribe/formDetails.ts

This is removed as a plug.

> src/components/Shifting/ShiftDetailsUpdate.tsx

This as well related to Model.

> src/components/Notifications/NotificationsList.tsx

Left out is the model field coming from backend

----


@rithviknishad what do you think ?

**Comment by yash-learner** [comment]
> **Actionable comments posted: 0**
> 
> 🧹 Outside diff range and nitpick comments (1)
> > src/components/Facility/Consultations/DailyRounds/DefaultLogUpdateCard.tsx (1)> Line range hint `1-15`: **Update import paths to reflect new terminology.**
> > > The import path still contains "DailyRounds" which should be updated to maintain consistency with the new "LogUpdate" terminology.
> > > ```diff
> > > - import LogUpdateCardAttribute from "@/components/Facility/Consultations/DailyRounds/LogUpdateCardAttribute";
> > > + import LogUpdateCardAttribute from "@/components/Facility/Consultations/LogUpdates/LogUpdateCardAttribute";
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > Note: This change would require moving the file to the new directory structure. Would you like me to help create a GitHub issue to track this refactoring task?
> 
> 📜 Review details

@rithviknishad Do you think this change is required as of now ?

File: src/common/constants.tsx [hunk 3]
```diff
@@ -653,12 +653,12 @@ export const NOTIFICATION_EVENTS: NotificationEvent[] = [
   },
   {
     id: "PATIENT_CONSULTATION_UPDATE_CREATED",
-    text: "Patient Consultation Update Created",
+    text: "Patient Log Update Created",
```
[the remark this answers is not in the document]
Comment (yash-learner) [reply src/common/constants.tsx:656]: @coderabbitai Addressed in https://github.com/ohcnetwork/care_fe/pull/9089/commits/79a3616983537cdd49254914d9686c691c073437, https://github.com/ohcnetwork/care_fe/pull/9089/commits/d2a2e054d40c55831dc76cc92b27239dbb66a069

**Comment by rithviknishad** [comment]
@yash-learner yes let's do that too

**Comment by yash-learner** [comment]
> @yash-learner yes let's do that too

@rithviknishad 

Fixed it in https://github.com/ohcnetwork/care_fe/pull/9089/commits/eccc86a375776fb3c7f8745e0c171649113cbe2e

File: src/components/Facility/Consultations/DailyRoundsList.tsx
Comment (rithviknishad) [inline src/components/Facility/Consultations/DailyRoundsList.tsx:1]: File could be renamed to `LogUpdatesList.tsx`; lgtm otherwise

Comment (yash-learner) [reply src/components/Facility/Consultations/DailyRoundsList.tsx:1]: Thanks, fixed in https://github.com/ohcnetwork/care_fe/pull/9089/commits/8d6edaaba07ba9a7a74f496c4d537b0badfe155c, https://github.com/ohcnetwork/care_fe/pull/9089/commits/54c6464eaabf10a58642948095e79dbaccf5507e

**Review by rithviknishad — APPROVED** [review]

**Pull request merged**